frontend/src/pages/org/settings/settings.ts

```diff
@@ -275,6 +272,19 @@ export class OrgSettings extends TailwindElement {
     </div>`;
   }
 
+  private handleSlugInput(e: InputEvent) {
+    const input = e.target as SlInput;
+    // Ideally this would match against the full character map that slugify uses
+    // but this'll do for most use cases
+    const end = input.value.match(/[\s*_+~.,()'"!\-:@]$/g) ? "-" : "";
+    input.value = slugifyStrict(input.value) + end;
+    this.slugValue = slugifyStrict(input.value);
+
+    input.setCustomValidity(
+      this.slugValue.length < 2 ? "URL Identifier too short" : "",
```

Comment:
I'm unsure if using default browser translations will translate custom validity messages?
```suggestion
      this.slugValue.length < 2 ? msg("URL Identifier too short") : "",
```

Reply:
Oh good catch, thank you!